konrad/convection.py

```diff
@@ -320,48 +392,36 @@ def get_convective_tau(self, p):
         if self.convective_tau is not None:
             return self.convective_tau
 
-        tau0 = 1/24 # 1 hour
+        tau0 = 1/24  # 1 hour
```

Comment:
Could this be a initialisation parameter? Or do you want to keep it simple ("Profile or nothing.") for now?

Reply:
I think it's okay as it is, at least for now. Someone might want to test the dependency of the profile / convective heating rates of `tau0`, but I think it would be more interesting to compare to a completely different profile for tau(p) or with a different dependency _e.g._ tau(T).